### DISCUSSION TITLE
how to change fdp

### DISCUSSION THREAD

**Discussion by algrowww** [SkidderMC/FDPClient#875] [General]
Hello sir, I hope you're doing well. I appreciate your work. However, I have encountered a problem. My friend from SMP (Survival Multiplayer) suggested that I share my screen with them, but since I'm using an FDP client, I am unable to do so. Therefore, I need your help with three things. Firstly, could you please guide me on how to change the icon? Secondly, I would like to know how I can change my name. Whenever I open the game, it displays my FDP name and a GitHub link. Lastly, I'm wondering how I can change the background to a normal one. I'm eagerly awaiting your response and I would be extremely grateful for your assistance. Thank you for your kindness in advance.

**Comment by B4mb0u** [comment]
lmaoo

**Reply by algrowww** [reply]
Yo How i can Contact With Developer?

**Comment by ghost** [comment]
Hello Yousaf1796! 
The name will be stored everywhere, so that the client is not your own, that will not be allowed just like that and the developers will not tell you that. Here is a tutorial how to change the background: https://youtu.be/5HhGqS0onHE
